examples/tfengine/team.hcl

```diff
@@ -118,6 +118,10 @@ EOF
       secrets = [
         {
           secret_id = "manual-sql-db-user"
+          locations = [
```

Comment:
To be consistent with others can we call this secret_locations?

Reply:
done